subsys/ipc/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Nordic Semiconductor (ASA)
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig IPC_SERVICE
```

Comment:
The name IPC_SERVICE is not that great, could this be tied to the implementation, i.e. OpenAMP? There are many ways IPC can be interpreted and implemented in an RTOS. The naming here is generic, the implementation is not.

Reply:
How about `RPMSG_SERVICE`? Would it be OK?

Reply:
yeah, for example. That would tell me exactly what it is.